plugins/modules/aci_bd.py

```diff
@@ -162,6 +254,7 @@
   link: https://developer.cisco.com/docs/apic-mim-ref/
 author:
 - Jacob McGill (@jmcgill298)
+- Samita Bhattacharjee (@samitab)
```

Comment:
Should we add an example with the new options?

Reply:
Done